### PR TITLE
OFFAPPS-622 just use the locale from the framework

### PR DESCRIPTION
**Pull request by sandlerr**
:koala:

/cc @zendesk/quokka

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -46,21 +39,19 @@
...
-      }).bind(this));
+      this.hasActivated = true;
+      this.currAttempt = 0;
+      this.storeUrl = this.storeUrl || this.checkStoreUrl(this.settings.url);
+      this.requiredProperties = [
+        'ticket.requester.email'
+      ];
+
+      if (this.currentLocation() === 'ticket_sidebar') {
+        this.allRequiredPropertiesExist();
```
Comment (danielbreves) [inline app.js]: We shouldn't need this logic anymore, this is legacy code from the time the container activated before the data API was fetched.

**Comment by sandlerr** [comment]
@danielbreves :haircut:

File: app.js [hunk 2]
```diff
@@ -46,21 +38,13 @@
...
-
-        if (this.currentLocation() === 'ticket_sidebar') {
-          this.allRequiredPropertiesExist();
-        } else if (this.ticket().requester()) {
-          // user may have selected a requester and reloaded the app
-          this.queryCustomer();
-        }
-      }).bind(this));
+      this.hasActivated = true;
+      this.currAttempt = 0;
```
Comment (danielbreves) [inline app.js]: This property is no longer used.

File: app.js [hunk 3]
```diff
@@ -46,21 +38,13 @@
       if(!data.firstLoad){
```
Comment (danielbreves) [inline app.js]: This should probably be changed to use 'app.created' now.

File: app.js [hunk 4]
```diff
@@ -46,21 +38,13 @@
...
-        ];
-
-        if (this.currentLocation() === 'ticket_sidebar') {
-          this.allRequiredPropertiesExist();
-        } else if (this.ticket().requester()) {
-          // user may have selected a requester and reloaded the app
-          this.queryCustomer();
-        }
-      }).bind(this));
+      this.hasActivated = true;
```
Comment (danielbreves) [inline app.js]: I don't think this guard is necessary anymore.

**Comment by sandlerr** [comment]
@danielbreves doing some more cleanup. Should I convert it to secure settings?

**Comment by danielbreves** [comment]
Nice :+1:

**Comment by danielbreves** [comment]
haha nah, this is fine.

**Comment by adammw** [comment]
wow :+1:

**Pull request merged**